canvas/cli/common.py

```diff
@@ -1,12 +1,19 @@
+import json
 import sys
 
 # keys: [(items key, title, pretty title), ...]
 def cli_list(items, keys, table = False, skip_headers = False,
-        collective_name = 'items', sort_key = 'id'):
+        collective_name = 'items', sort_key = 'id',
+        **kwargs):
     if (len(items) == 0):
         print("No %s found." % (collective_name), file = sys.stderr)
         return 0
 
+    is_json = kwargs.get('json', False)
```

Comment:
Instead of pulling values out of kwargs,
it is more idiomatic to just list it as a named arg.
Remember that kwargs will capture all the unlisted named args,
so you will have access to the arg as long as you list it.